src/lib/components/Creatibutors/CreatibutorsModal.js

```diff
@@ -196,11 +197,48 @@ export class CreatibutorsModal extends Component {
     }
   };
 
+
+  makeIdEntry = (identifier) => {
+
+    let icon = null;
+    let link = null;
+    let linkTitle = null;
+
+    if (identifier.scheme == "orcid") {
+      icon = "/static/images/orcid.svg";
+      link = "https://orcid.org/" + identifier.identifier;
+    } else if (identifier.scheme == "gnd") {
+      icon = "/static/images/gnd-icon.svg";
+      link = "https://d-nb.info/gnd/" + identifier.identifier;
+    } else if (identifier.scheme == "ror") {
+      icon = "/static/images/ror-icon.svg";
+      link = "https://ror.org/"+identifier.identifier;
+    } else {
+      return (<>{identifier.scehme}: {identifier.identifier}</>)
+    }
+
+    return (
+      <>
+        <Image
+          src={icon}
+          className="inline-id-icon"
+          verticalAlign="middle"
+          key={identifier.identifier}
+        />
+        {identifier.identifier}
+          <a
+            href={link}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <Icon link name="external alternate" className="spaced-left" />
```

Comment:
```suggestion
            <Icon link name="external alternate" className="ml-5" />
```

`.spaced-left` does not exist

Reply:
Done.